RELEASE_NOTES.rst

```diff
@@ -1,3 +1,132 @@
+  `cEP-0005 <https://github.com/coala/cEPs/blob/master/cEP-0005.md>`_
+  for a syntax example and `PR #3178 <https://github.com/coala/coala/pull/3178/>`_
+  for more information.
+- The section name ``cli`` is reserved for cli arguments and should not be
+  used in the coala configuration file (``coafile``). If you are currently
+  using it in a configuration file, coala will warn you.
+- coala now outputs absolute file paths when a bear fails. This should
+  make debugging bear failures easier when using custom bears.
+
+**Usability**
+
+- A new placeholder, ``source_lines``,  has been added to the coala
+  ``--format`` option. Use it to output the affected lines in the source file
+  when a result occurs.
+- Bears are displayed alphabetically when listed using ``--show-bears``.
+- coala outputs a warning if a setting is overridden in the same section
+  since this is probably not what the user intended.
+- ``--format`` is handled correctly when ``--ci`` is specified, outputting
+  formatted output without asking questions.
+  `Issue #3999 <https://github.com/coala/coala/issues/3999>`_
+- coala will report the version of the dependency required when the version
+  check fails.
+  `Issue #3982 <https://github.com/coala/coala/issues/3982>`_
+
+**Bear Developers**
+
+- ``@linter`` decorator works with global bears. See our bear writing
+  `docs <http://api.coala.io/en/latest/Developers/Writing_Linter_Bears.html#global-bears>`__
+  for more information on how to write a global bear with the ``@linter``
+  decorator.
+- ``Bear.DownloadCachedFile`` raises exceptions when the download fails.
+  Please handle this exception if the file download has a good chance
+  of failing.
+- The ``deprecate_settings`` decorator now supports multiple deprecations
+  of the same setting.
+  `Issue # 3537 <https://github.com/coala/coala/issues/3537>`_
+- A new ``SEE_MORE`` attribute has been added to the ``Bear`` class. Use it
+  to attribute (usually by linking the main page)  the original tool in
+  linter bears.
+- Unified Diff support has been added to the ``Diff`` result, linter
+  and display mechanisms.
+
+**Bugfixes**
+
+- Caching has been fixed to work correctly. Results are only cached when a
+  bear successfully runs so a bear which fails in the middle of a run will 
+  not have a result cached anymore. This should make flushing the cache 
+  manually mostly unnecessary for end users.
+  `PR #3722 <https://github.com/coala/coala/pull/3722>`_
+- Non-optional settings check settings from dependent bears now. Previously,
+  it would error because the dependency would not have its settings satisfied.
+  `Issue #3149 <https://github.com/coala/coala/pull/3149>`_
+- The ``Results`` object now handles ``None`` values of TextPosition properly.
+  Previously, it would return arbitrary results which would impact bear
+  authors.
+  `Issue #3094 <https://github.com/coala/coala/issues/3094>`_
+- A few locales problems when using coala in Docker have been fixed.
```

Comment:
the cause is a system with incomplete locale data, which is most common on docker images which omit recommended packages.